test/cosmoz-bottom-bar-view.html

```diff
@@ -87,7 +87,7 @@
 				bottomBarView = fixture('scrollingBottomBarViewFixture');
 				bottomBar = bottomBarView.$.bottomBar;
 				flushBottomBar(bottomBar, function (){
-					Polymer.Base.async(done, 50);
+					Polymer.Base.async(done, 100);
```

Comment:
Use setTimeout instead here?

Reply:
Yes, was going to save it for next pull request, but fixed it now.

Reply:
Ah, ok!